packages/rlc-common/src/metadata/buildPackageFile.ts

```diff
@@ -174,10 +181,19 @@ function restLevelPackage(model: RLCModel) {
       prettier: "^3.1.0",
       rimraf: "^5.0.0",
       "source-map-support": "^0.5.9",
-      typescript: "~5.3.3"
+      typescript: "~5.3.3",
+      ...(model.options?.moduleKind === "esm" && { tshy: "1.11.1" })
     }
   };
 
+  if (model.options?.moduleKind === "esm") {
+    packageInfo.scripts["build"] =
+      "npm run clean && tshy && npm run extract-api";
+  } else {
+    packageInfo.scripts["build"] =
+      "npm run clean && tsc && rollup -c 2>&1 && npm run minify && mkdirp ./review && npm run extract-api";
```

Comment:
is it for non-branded only ? I wonder what's the difference between line 194 and line 220-221 ?